api/apps/api/test/access-control/project-acl-get.e2e-spec.ts

```diff
@@ -0,0 +1,132 @@
+import { FixtureType } from '@marxan/utils/tests/fixture-type';
+import { getFixtures } from './project-acl.fixtures';
+
+let fixtures: FixtureType<typeof getFixtures>;
+
+beforeEach(async () => {
+  fixtures = await getFixtures();
+});
+afterEach(async () => {
+  await fixtures?.cleanup();
+});
+
+test(`getting project users when user is not added to project`, async () => {
```

Comment:
very minor, but the "added" bit feels off
```suggestion
test(`getting project users when user has no role in the project`, async () => {
```